public/components/data_connections/components/acceleration_ui/acceleration_header.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiLink,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React from 'react';
+import { OPENSEARCH_DOCUMENTATION_URL } from '../../../../../common/constants/integrations';
+
+export const AccelerationHeader = () => {
+  return (
+    <div>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l" data-test-subj="acceleration-header">
+            <h1>Acceleration Indices</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiSpacer size="s" />
+      <EuiText size="s" color="subdued">
+        Manage acceleration indices from external data connections.{' '}
+        <EuiLink external={true} href={OPENSEARCH_DOCUMENTATION_URL} target="blank">
```

Comment:
i think we mostly use `_blank`, otherwise different links will replace previous page in the same new tab

Reply:
Thanks for the catch!